src/envs/stick_point.py

```diff
@@ -111,7 +111,10 @@ def simulate(self, state: State, action: Action) -> State:
         # Actions are normalized to [-1, 1]. Denormalize them here.
         dx = norm_dx * self.max_speed
         dy = norm_dy * self.max_speed
-        dtheta = norm_dtheta * self.max_angular_speed
+        if CFG.stick_point_disable_angles:
+            dtheta = 0.0
+        else:
+            dtheta = norm_dtheta * self.max_angular_speed
         # Update the robot state.
         rx = state.get(self._robot, "x")
```

Comment:
in the example video you showed, the arrow looks a little silly / out of place. can we just not show the arrow when `CFG.stick_point_disable_angles`?